airgun/views/job_invocation.py

```diff
@@ -56,9 +56,7 @@ class target_hosts_and_inputs(WizardStepView):
 
         targetting_type = Select(locator='//div[button[@aria-haspopup="listbox"]]')
         # https://bugzilla.redhat.com/show_bug.cgi?id=2265127
```

Comment:
This BZ is resolved now so can you remove this?
This change affected the locator, and it is currently being worked by @lhellebr  - https://issues.redhat.com/browse/SAT-23424

Reply:
Removed suggested line

Reply:
@amolpati30 Not `targetting_type` locator, just remove the bugzilla comment :)